sdk/cognitiveservices/Knowledge.QnAMaker/src/Customizations/EndpointKeyServiceClientCredentials.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker
```

Comment:
Does this need a copyright header?

Reply:
Kept it same as ApiKeyServiceClientCredentials in the same folder. I can add MIT header. Let me know.

Reply:
added